src/sctools/count.py

```diff
@@ -90,6 +90,41 @@ def _get_alignments_grouped_by_query_name_generator(
                 )
                 yield query_name, cell_barcode, molecule_barcode, alignments
 
+    """Looks through a list of gene locations to find the one that the given read_start ovelaps
+
+    Parameters
+    ----------
+    gene_locations: Array
+        array with gene start end locations and names
+    search_start:
+        index of gene to start searching from
+    search_end:
+        index of gene up to which to search to
+    read_start:
+        position at which the read starts at
+
+    Returns
+    -------
+        name of gene with overlap or None if no overlap is found
+
+    """
+
+    @classmethod
+    def binary_overlap(cls, gene_locations, search_start, search_end, read_start):
+        while search_start <= search_end:
+            current_gene_index = int((search_start + search_end) / 2)
+            if (
+                gene_locations[current_gene_index][0][0]
```

Comment:
Please rename variable for readability

Reply:
added and _ to distinguish this hidden internal variable in accordance with PEP8 convention